service/test_fixtures/psql/0_initial_schemas.up.sql

```diff
@@ -0,0 +1,98 @@
+CREATE TYPE permission_level AS ENUM
+(
+  'NO_PERM',
+  'USER_REGISTRATION',
+  'USER',
+  'ADMIN'
+);
+
+-- https://github.com/oklog/ulid
+CREATE DOMAIN ulid AS
+  VARCHAR(26) NOT NULL CHECK (LENGTH(VALUE) = 26);
+
+
+
+
+CREATE SCHEMA user_svc;
+
+CREATE DOMAIN user_svc.user_name AS
+  VARCHAR(32) NOT NULL CHECK (VALUE ~ '^[[:alpha:]]+(([''.\s-][[:alpha:]\s])?[[:alpha:]]*)*$');
+
+-- https://github.com/segmentio/ksuid
+CREATE DOMAIN user_svc.ksuid AS
+  VARCHAR(27) NOT NULL CHECK (LENGTH(VALUE) = 27);
+
+CREATE TABLE user_svc.accounts
+(
+  uuid              ulid PRIMARY KEY,
+  first_name        user_svc.user_name,
+  last_name         user_svc.user_name,
+  email             VARCHAR(320) NOT NULL UNIQUE,
```

Comment:
Can you make another migration source code to make `email` an index?

`CREATE INDEX user_svc_accounts_email_idx ON user_svc.accounts(email);`